fe/fe-core/src/main/java/com/starrocks/sql/common/SyncPartitionUtils.java

```diff
@@ -389,4 +396,41 @@ public static Range<PartitionKey> createRange(String lowerBound, String upperBou
                 Collections.singletonList(partitionColumn));
         return Range.closedOpen(lowerBoundPartitionKey, upperBoundPartitionKey);
     }
+
+    public static void dropBaseVersionMeta(MaterializedView mv, String basePartitionName) {
+        MaterializedView.AsyncRefreshContext refreshContext = mv.getRefreshScheme().getAsyncRefreshContext();
+        Map<Long, Map<String, MaterializedView.BasePartitionInfo>> versionMap =
+                refreshContext.getBaseTableVisibleVersionMap();
+        if (versionMap == null) {
+            return;
+        }
+        Expr expr = mv.getPartitionRefTableExprs().get(0);
+        SlotRef slotRef;
+        if (expr instanceof SlotRef) {
```

Comment:
keep the same with line 426? I think they are the same?
add some comment for this condition